### PR TITLE
Remove unneeded backgrounds in intro-login stack

### PR DESCRIPTION
**Pull request by rasom**
### BEFORE/AFTER

---

---

---

---

---

### CODE REVIEW

**Review by yenda — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by rasom** [comment]
We can't merge this PR till react-navigation-stack will be properly forked and included into our project (otherwise some backgrounds will be gray instead of white)

**Comment by rasom** [comment]
can be merged after https://github.com/status-im/status-react/pull/7618

**Review by mandrigin — APPROVED** [review]

**Pull request merged**
